libs/sdk-core/src/swap.rs

```diff
@@ -408,7 +395,7 @@ impl BTCReceiveSwap {
                     amount_sats: swap_info.confirmed_sats,
                     description: String::from("Bitcoin Transfer"),
                     preimage: Some(swap_info.preimage),
-                    opening_fee_params: swap_info.channel_opening_fees,
+                    opening_fee_params: None,
```

Comment:
Shouldn't this be passed in here?

Reply:
I checked, this is based on a previous commit. The param is currently passed correctly.